ftw/testbrowser/pages/editbar.py

```diff
@@ -0,0 +1,146 @@
+
+
+@QueryInfo.build
+def menu(label, browser=default_browser, query_info=None):
+    """Finds a menu by label and returns its ``<dl class="actionMenu">`` node.
+
+    :param label: The label of the menu to find.
+    :type label: string
+    :param browser: The browser instance to operate with. Uses the global singleton
+      default browser by default.
+    :type browser: :py:class:`ftw.testbrowser.core.Browser`
+    :returns: The menu container node.
+    :rtype: :py:class:`ftw.testbrowser.nodes.NodeWrapper`
+    :raises: :py:exc:`ftw.testbrowser.exceptions.NoElementFound`
+    """
+    menus_node = container(browser=browser).css('#contentActionMenus').first
+    label = normalize_spaces(label).rstrip(u'\u2026')
+    for span in menus_node.css('.actionMenuHeader > a > span:first-child'):
+        if normalize_spaces(span.text_content()).rstrip(u'\u2026') == label:
+            return span.parent('.actionMenu')
+
+    query_info.add_hint('Visible menus: {!r}.'.format(menus(browser=browser)))
+    raise NoElementFound(query_info)
+
+
+def menu_options(menu_label, browser=default_browser):
+    """Returns the labels of the options of a menu.
+
+    :param menu_label: The label of the menu to find.
+    :type menu_label: string
+    :param browser: The browser instance to operate with. Uses the global singleton
+      default browser by default.
+    :type browser: :py:class:`ftw.testbrowser.core.Browser`
+    :returns: A list of the labels of the visible menus.
+    :rtype: list of str
+    """
+    menu_container = menu(menu_label, browser=browser)
+    return menu_container.css('.actionMenuContent a .subMenuTitle').text
+
+
+@QueryInfo.build
+def menu_option(menu_label, option_label, browser=default_browser,
+                query_info=None):
+    """Returns the link node (``<a>``) of an option in a menu.
+
+    :param menu_label: The label of the menu.
+    :type menu_label: string
+    :param menu_label: The label of the option to find in the menu.
+    :type menu_label: string
+    :param browser: The browser instance to operate with. Uses the global singleton
+      default browser by default.
+    :type browser: :py:class:`ftw.testbrowser.core.Browser`
+    :returns: The option link node.
+    :rtype: :py:class:`ftw.testbrowser.nodes.NodeWrapper`
+    :raises: :py:exc:`ftw.testbrowser.exceptions.NoElementFound`
+    """
+
+    menu_container = menu(menu_label, browser=browser, query_info=query_info)
+    option_label = normalize_spaces(option_label)
+    for link in menu_container.css('.actionMenuContent a'):
+        if normalize_spaces(link.text_content()) == option_label:
+            return link
+
+    query_info.add_hint('Options in menu {!r}: {!r}'.format(
+        menu_label,
+        menu_options(menu_label, browser=browser)))
+    raise NoElementFound(query_info)
+
+
+def container(browser=default_browser):
+    """Returns the editbar container node or ``None`` if it is not visible.
```

Comment:
Does this really return None, when its not visible ... 🤔  Your test say it raises `NoElementFound`